databricks_cli/configure/cli.py

```diff
@@ -38,6 +40,15 @@
 PROMPT_PASSWORD = 'Password'  # NOQA
 PROMPT_TOKEN = 'Token'  # NOQA
 ENV_AAD_TOKEN = 'DATABRICKS_AAD_TOKEN'
+DEFAULT_SCOPES = [
```

Comment:
Nit: I still think these scopes are a little weird to include in the CLI as I'm not sure there is like an API that uses SCIM here (I think it uses the Groups API, which is pretty unintuitive).

Reply:
Good call. Removed SCIM here.